exercises/container-security/docker-hub-cli/test/getDigest.test.js

```diff
@@ -0,0 +1,47 @@
+const { validateDigestArgs, getDigest } = require('../cli/index');
+const { getDockerContentDigest } = require('../lib/index.js');
+
+describe('validate args', () => {
```

Comment:
Maybe name directly after test function `validateDigestArgs` or `validateDigestArgs()`? Also makes sense it is named consistently to test name in L23.